api/app/src/command/webhook/garmin.ts

```diff
@@ -25,7 +25,7 @@ type WebhookDataPayload = {
   users: WebhookUserPayload[];
 };
 type WebhookDataPayloadWithoutMessageId = Omit<WebhookDataPayload, "meta">;
-type WebhookUserPayload = { userId: string } & WebhookUserDataPayload;
+type WebhookUserPayload = { id: string } & WebhookUserDataPayload;
```

Comment:
we don't need to change this name in the payload - let's just leave as `userId`

Reply:
I suggest we send both... whoever is using `userId` can keep using it as such (cx's user id) and who jumps in or wants to use Metriport's user id can do it intentionally.

Changing the value without changing the prop name is a silent change that is likely to break stuff. 😄

Reply:
Can mark the older one as deprecated and remove from docs, and create/schedule a ticket to remove it on a major version update that users get notified about beforehand.

Reply:
See if my changes meet the requirements for deprecation. I wasn't totally sure how/where to mark it as such.